server/aesdsocket-start-stop.sh

```diff
@@ -0,0 +1,17 @@
+#! /bin/sh
```

Comment:
@PeterBraganza you shouldn't check your server/aesdsocket file into your repo - you should build this with buildroot.